processing/src/main/java/io/druid/segment/incremental/IncrementalIndexStorageAdapter.java

```diff
@@ -590,6 +591,12 @@ public Object get()
 
                 return null;
               }
+
+              @Override
+              public ValueType columnType(String columnName)
+              {
+                return index.getMetricValueType(columnName);
```

Comment:
Currently it returns null. I think I should recheck with dimensions for consistency.